internal/cli/doctor.go

```diff
@@ -35,22 +35,45 @@ func (c *doctorCmd) run(ctx context.Context, args []string) error {
 	fmt.Println("---")
 	fmt.Println("Docker")
```

Comment:
Oh wait this should def be `Docker (cluster)` to differentiate from `Docker (local)`

Reply:
done!